contracts/TokenRequest.sol

```diff
@@ -153,8 +219,29 @@ contract TokenRequest is AragonApp {
         }
 
         tokenManager.mint(requesterAddress, requestAmount);
+        uint64 finalizedDate = getTimestamp64();
 
-        emit TokenRequestFinalised(requesterAddress, depositToken, depositAmount, requestAmount);
+        emit TokenRequestFinalised(_tokenRequestId, requesterAddress, depositToken, depositAmount, requestAmount, finalizedDate);
```

Comment:
As above, there is no need to record the time this event is emitted, as it's stamped in the block chain for us.